client/my-sites/plan-features/style.scss

```diff
@@ -386,6 +386,19 @@ $plan-features-sidebar-width: 272px;
 	text-align: center;
 }
 
+.ribbon__wrapper.plan-features__header-ribbon {
```

Comment:
I am explicitly reading `className` prop here: https://github.com/Automattic/wp-calypso/pull/6793/files#diff-586334d796bbeeefbfafd30bc436b544R7
Or you would like it more inline?